stable/prometheus-operator/ci/test-values.yaml

```diff
@@ -885,7 +885,7 @@ kube-state-metrics:
 ## Deploy node exporter as a daemonset to all nodes
 ##
 nodeExporter:
-  enabled: true
+  enabled: false
```

Comment:
This change should not be included in the PR, it prevents testing configurations of the node-exporter sub-chart

Reply:
I have re-enabled the node-exporter in the test values however this is likely to break the tests again due to the pod placement issues on the test cluster.

If the chart test is failing will that block merging this change?